resources/views/product/partials/images.blade.php

```diff
@@ -1,73 +1,33 @@
+                    <x-heroicon-o-chevron-left class="h-8 w-8 text-gray-900" />
                 </button>
-
-                <button class="top-1/2 -translate-y-1/2 right-3 {{ config('rapidez.z-indexes.lightbox')}}" :class="zoomed ? 'fixed' : 'absolute'" v-if="active != images.length-1" v-on:click="change(active+1)" aria-label="@lang('Next')">
-                    <x-heroicon-s-chevron-right class="h-12 w-12"/>
+                <button class="{{ config('rapidez.z-indexes.lightbox') }} top-1/2 right-3 -translate-y-1/2" :class="zoomed ? 'fixed' : 'absolute'" v-if="active != images.length-1" v-on:click="change(active+1)" aria-label="@lang('Next')">
+                    <x-heroicon-o-chevron-right class="h-8 w-8 text-gray-900" />
                 </button>
             </div>
-            <x-rapidez::no-image v-else class="rounded h-96"/>
+            <x-rapidez::no-image v-else class="h-96 rounded" />
 
-            <div v-if="images.length > 1" class="flex" :class="zoomed ? 'fixed bottom-0 left-3 {{ config('rapidez.z-indexes.lightbox')}}' : ' flex-wrap mt-3'">
-                <a
-                    href="#"
-                    v-for="(image, imageId) in images"
-                    class="flex items-center justify-center bg-white border rounded p-2 mr-3 mb-3 h-[100px] w-[100px]"
-                    :class="active == imageId ? 'border-primary' : ''"
-                    @click.prevent="change(imageId)"
-                >
-                    <img
-                        :src="'/storage/resizes/80x80/catalog/product' + image + '.webp'"
-                        alt="{{ $product->name }}"
-                        class="object-contain w-full m-auto max-h-[80px]"
-                        loading="lazy"
-                        width="80"
-                        height="80"
-                    />
+            <div v-if="images.length > 1" class="flex" :class="zoomed ? 'fixed bottom-0 left-3 {{ config('rapidez.z-indexes.lightbox') }}' : ' flex-wrap mt-3'">
+                <a href="#" v-for="(image, imageId) in images" class="mr-3 mb-3 flex h-[100px] w-[100px] items-center justify-center rounded border bg-white p-2" :class="active == imageId ? 'border-primary' : ''" @click.prevent="change(imageId)">
+                    <img :src="'/storage/resizes/80x80/catalog/product' + image + '.webp'" alt="{{ $product->name }}" class="m-auto max-h-[80px] w-full object-contain" loading="lazy" width="80" height="80" />
                 </a>
             </div>
-
-            <div v-if="zoomed" class="fixed top-3 right-3 pointer-events-none {{ config('rapidez.z-indexes.lightbox')}}">
-                <x-heroicon-o-x class="h-6 w-6"/>
+            <div v-if="zoomed" class="{{ config('rapidez.z-indexes.lightbox') }} pointer-events-none fixed top-3 right-3">
```

Comment:
My preference with merging php variables into classes would be.
Keep in mind this does not get recognised by tailwind and might not get compiled if the class is not used somewhere else
```suggestion
            <div v-if="zoomed" @class([
            	config('rapidez.z-indexes.lightbox'), 
            	'pointer-events-none fixed top-3 right-3'
            ])>
```